### ISSUE TITLE
The telemetry is not displayed in Glean Debug Ping Viewer if using iOS Simulators.

### ISSUE THREAD

**Issue by AlexandraBerar** [mozilla-mobile/firefox-ios#15043]
[Affected versions]:
- All of the versions of Firefox builds.

[Affected Platforms]:
- iOS simulated devices

[Prerequisites]:
- Have a simulator up and running.
- The user has all the necessary setup to make a Firefox build on Prod or stage server.

[Steps to reproduce]:

1. Using Xcode run a Firefox build in the iOS simulator.
2. If the Firefox build is opened after starting the simulator, close it.
3. Navigate to Safari app from the simulator and paste the link from the previously created glean (e.g: firefox://glean?logPings=true&sendPing=events&debugViewTag=alexandraDefault)
4. Make an action using the Firefox app.
5. Send the Firefox app in the background and observe the pings which are registered on: https://debug-ping-preview.firebaseapp.com/

**[Expected results]:**
The pings’ events should be registered on debug-ping even if the user is using a simulated device.

**[Actual results]:**
The pings’ events are not registered on debug-ping if the user is using a simulator.

[Additional notes]:
The pings’ events are registered if the user is using a real device.
Here is a screen recording of the issue: [Gdoc](https://drive.google.com/file/d/1VdCqh850wQ4MOqK_EayDmz87wGJjTakZ/view?usp=sharing).

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-6758)

**Comment by abodea** [comment]
Hello, please note that I was able to generate telemetry in the Glean debug pinger, https://debug-ping-preview.firebaseapp.com/pings/Shamy23 here are all the events I just triggered on the simulator.
@AlexandraBerar can you please try to do any action in App and after that to add FF to background? The telemetry should trigger after adding FF to background.

**Comment by abodea** [comment]
Hello, @Dexterp37 if you have time can you please take a look, somehow on my end it's working but @AlexandraBerar and @SoftVision-CiprianMuresan can't make it work.

**Comment by rosahbruno** [comment]
@AlexandraBerar @SoftVision-CiprianMuresan 
Hello!

Is there a specific target/build you are running that you are seeing issues with (Fennec, Firefox, FirefoxBeta, etc)? I was able to run and see debug pings with the "Fennec" target without any issues.

A few call outs about Glean debugging and the debug ping viewer
1. Sometimes there are delays when using the debug ping viewer. Pings typically show up within a few seconds, but there are times where pings won't be received for (potentially) a few minutes.
2. Another option for debugging is to [set an environment variable `GLEAN_DEBUG_VIEW_TAG`](https://mozilla.github.io/glean/book/reference/debug/debugViewTag.html#glean_debug_view_tag).

I was able to run the app using both of the debugging options (deeplink and environment variable) and see pings showing up in the debug ping viewer.
